src/Common/src/Obsoletions.cs

```diff
@@ -21,4 +21,9 @@ internal static class Obsoletions
     internal const string DomainItemAccessibleObjectMessage = $"{nameof(DomainUpDown.DomainItemAccessibleObject)} is no longer used to provide accessible support for {nameof(DomainUpDown)} items.";
 #pragma warning restore WFDEV003 // Type or member is obsolete
     internal const string DomainItemAccessibleObjectDiagnosticId = "WFDEV003";
+
+#pragma warning disable WFDEV004
+    internal const string ListBoxDefaultItemHeightMessage = $"The {nameof(ListBox.DefaultItemHeight)} const is replaced with the ListBox.DefaultListBoxItemHeight property.";
```

Comment:
Referring to a private property may not be helpful for those who will want to replace the const in their code. I suggest mentioning what can be used instead and also saying that default height is now dependent on application's font.

```suggestion
    internal const string ListBoxDefaultItemHeightMessage = $"{nameof(ListBox.DefaultItemHeight)} is no longer used as default item height, which now depends on application's default font. Use {nameof(ListBox.ItemHeight)} instead.";
```

Reply:
Yes, that's a good point here.